src/styles/annotator/pdfjs-overrides.scss

```diff
@@ -15,3 +15,13 @@
 .textLayer .highlight.selected {
   background-color: rgba(0, 100, 0, 0.5);
 }
+
+// Remove horizontal `margin:auto` so that PDF pages are left-aligned
```

Comment:
Hmm, I wonder whether we might want to apply this only when side-by-side mode is actually active. It looked odd to me when the sidebar is closed. The downside of conditionally applying it is that the content will "jump" more when the sidebar opens, but I think that may be better than having it off-center when trying to read rather than annotate.

Reply:
I've reworked this in my latest round of changes.